### PR TITLE
Fix links to anchors

### PR DESCRIPTION
**Pull request by abhishek-raj**
The anchors created from the markdown are in lower case. I changed all links to anchors to lower case.

### CODE REVIEW

**Comment by abhishek-raj** [comment]
Seems like we aren't using the translation features of docusaurus and thus the links in the footer isn't working. Fixed those too.

File: website/core/Footer.js [hunk 1]
```diff
@@ -27,35 +27,35 @@ class Footer extends React.Component {
               href={
                 this.props.config.baseUrl +
                 'docs/' +
-                this.props.language +
-                '/getting-started.html'
+                // this.props.language +
```
Comment (brentvatne) [inline website/core/Footer.js]: we should leave this in. it produces a slightly weird url right now but it still works, and we'll add translations shortly

**Comment by abhishek-raj** [comment]
Reverted back the changes to Footer.js.

File: website/core/Footer.js (hunk 1 above)
Comment (abhishek-raj) [reply website/core/Footer.js]: Okay. Reverted back the Footer.js

**Comment by brentvatne** [comment]
thanks!

**Pull request merged**